### DISCUSSION TITLE
VIA definitions for manual download

### DISCUSSION THREAD

**Discussion by exen904** [vial-kb/vial-gui#96] [Q&A]
Is it possible to get the VIA definitions as manual download option? In some corporate environments with a web proxy in front of the client, the download via the Button in "File" "Download Via definitions" doesnt work.

**Comment by LukeDRussell** [comment]
I just got them from [VIAs repo](https://github.com/the-via/keyboards/tree/master/src/keebio/iris). It could be your web proxy is still going to block the download.
